crates/tabby/src/services/completion.rs

```diff
@@ -109,19 +109,38 @@ pub struct Segments {
     suffix: Option<String>,
 
     /// The relative path of the file that is being edited.
-    /// When git_url is set, this is the path of the file in the git repository.
-    /// When git_url is empty, this is the path of the file in the workspace.
+    /// - When `git_url` is set, this is the path of the file in the git repository.
+    /// - When `git_url` is empty, this is the path of the file in the workspace.
     filepath: Option<String>,
 
     /// The remote URL of the current git repository.
     /// Leave this empty if the file is not in a git repository,
     /// or the git repository does not have a remote URL.
     git_url: Option<String>,
 
+    /// The relevant declaration code snippets provided by editor.
+    /// The editor could invoke the language server to get relevant declarations.
```

Comment:
```suggestion
    /// The relevant declaration code snippets provided by editor.
    /// It'll contains declarations extracted from `prefix` segments using LSP.
```

Reply:
But declarations is extracted from other files, not from the prefix.